chef/cookbooks/resolver/recipes/default.rb

```diff
@@ -29,6 +29,7 @@
 end
 
 dns_list << node[:dns][:nameservers]
+dns_list = dns_list.flatten.sort.uniq
```

Comment:
@dirkmueller but the sort is kind of wrong there, as what is in `nameservers` should definitely come after the nodes with dns-server role.